modules/ossm-threescale-integrate-1x.adoc

```diff
@@ -24,7 +24,11 @@ To configure the 3scale Istio Adapter, refer to {ProductName} custom resources f
 
 [NOTE]
 ====
-Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the service ID of the API you want to manage.
+Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the `service_id` of the API you want to manage. For example:
+
+* Create one handler per service by attaching service IDs to workloads by labeling the relevant Deployment.
+* Create multiple different `service_id` workloads under the same 3scale account that refer to a single handler. This stores the 3scale credentials in their normal fields except `service_id`.
+* Then you only need one handler per 3scale account, rather than one handler per service of each 3scale account.
```

Comment:
I'd rewrite this as follows (also consider this change for the identical text in ossm-threescale-integrate.adoc):

```suggestion
Pay particular attention to the `kind: handler` resource. You must update this with your 3scale account credentials. You can optionally add a `service_id` to a handler, but this is kept for backwards compatibility only, since it would render the handler only useful for one service in your 3scale account, and enabling 3scale for other services will require for you to create more handlers which only differ in their `service_id`s.

You are instead encouraged to use a single handler per 3scale account by following the steps below:

1. Create a handler for your 3scale account and specify your account credentials. Please omit any service identifier.
2. Edit or patch the Deployment resource of any services belonging to your 3scale account as follows:
  a. Add the `"service-mesh.3scale.net/service-id"` label with a value corresponding to a valid `service_id`.
  b. Add the `"service-mesh.3scale.net/credentials"` label with its value being the _name of the handler resource_ from step 1.
3. Whenever you further add services, perform step 2 to link it to your 3scale account credentials and to its service identifier.
```

Reply:
Thanks, @unleashed 

How does this affect the Handler configuration example and the Rule configuration example that follows?

Reply:
The service_id entry would disappear from the params in the handler and instead the user would label their workload as specified in the steps above, so workloads would need to have two labels: one to point to the handler so the adapter knows how to contact their 3scale administration portal and account, and the other label to specify the service identifier within their account that that particular workload represents.

Reply:
@unleashed Thanks for your reply. I've made some updates. Hopefully they are as you described or we are close to it :smile:

Reply:
I think it is there already, just added a minor remark on some wording to avoid confusion: https://github.com/openshift/openshift-docs/pull/32951#discussion_r647637711